doc/changes/changes-3.0.3.md

```diff
@@ -0,0 +1,11 @@
+# Exasol Virtual Schema Common 3.0.2, released 2020-07-06
```

Comment:
```suggestion
# Exasol Virtual Schema 3.0.3, released 2020-??-??
```

Reply:
Please, change the version in the pom file too, but don't release it yet, I also have something to do here, we'll release it later

Reply:
I also have additional change (on a separate branch)